XML/TestCases/BVT.xml

```diff
@@ -1,27 +0,0 @@
-<TestCases>
-    <test>
-        <testName>BVT-VERIFY-BOOT-ERROR-WARNINGS</testName>
-        <testScript>BVT-VERIFY-BOOT-ERROR-WARNINGS.py</testScript>
-        <files>.\Testscripts\Linux\azuremodules.py,.\Testscripts\Linux\BVT-VERIFY-BOOT-ERROR-WARNINGS.py,.\XML\Other\ignorable-boot-errors.xml</files>
-        <setupType>OneVM</setupType>
-        <Platform>Azure,WSL</Platform>
-        <Category>BVT</Category>
-        <Area>BVT</Area>
-        <Tags>provision</Tags>
-        <Priority>0</Priority>
-    </test>
-    <test>
-        <testName>BVT-PASSWD-POLICY-TEST</testName>
-        <testScript>BVT-PASSWD-POLICY-TEST.py</testScript>
```

Comment:
This case is removed?

Reply:
> This case is removed?

Yes, test is not doing anything relevant to our testing.

Reply:
it only run against RHEL for password policy check

Reply:
> it only run against RHEL for password policy check

test is irrelevant, it will exit with PASS on any other distro, and we're wasting a deployment to verify password which is given by the distro provided, that is not our use-case.